yoti-sdk-sandbox/src/main/java/com/yoti/api/client/sandbox/profile/request/attribute/SandboxAnchor.java

```diff
@@ -0,0 +1,103 @@
+package com.yoti.api.client.sandbox.profile.request.attribute;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.TimeZone;
+
+import com.yoti.api.client.Date;
+import com.yoti.api.client.DateTime;
+import com.yoti.api.client.Time;
+import com.yoti.api.client.spi.remote.util.AnchorType;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SandboxAnchor {
+
+    private final String type;
+    private final String value;
+    private final String subType;
+    private final long timestamp;
+
+    private SandboxAnchor(String type, String value, String subType, long timestamp) {
+        this.type = type;
+        this.value = value;
+        this.subType = subType;
+        this.timestamp = timestamp;
+    }
+
+    public static SandboxAnchor.SandboxAnchorBuilder builder() {
+        return new SandboxAnchor.SandboxAnchorBuilder();
+    }
+
+    @JsonProperty("type")
+    public String getType() {
+        return type;
+    }
+
+    @JsonProperty("value")
+    public String getValue() {
+        return value;
+    }
+
+    @JsonProperty("sub_type")
+    public String getSubType() {
+        return subType;
+    }
+
+    @JsonProperty("timestamp")
+    public long getTimestamp() {
+        return timestamp;
+    }
+
+    public static class SandboxAnchorBuilder {
```

Comment:
if we are going to expose the builder only thought 
```  
  public static SandboxAnchor.SandboxAnchorBuilder builder() {
        return new SandboxAnchor.SandboxAnchorBuilder();
  }
```
I think that this should be `private`

Reply:
But if the builder class is _private_, I can't use it outside of _SandboxAnchor_.

Reply:
Aaaaand you are right! I'll hide the comment xD